kaldb/src/test/java/com/slack/kaldb/chunkManager/IndexingChunkManagerTest.java

```diff
@@ -316,24 +334,49 @@ public void testAddMessage() throws Exception {
 
   private void testChunkManagerSearch(
       ChunkManager<LogMessage> chunkManager,
+      List<String> chunkList,
```

Comment:
I think this should be called `chunkIds` ?

Reply:
Done.